recipes/json_dto/all/conanfile.py

```diff
@@ -16,51 +21,52 @@ class JsondtoConan(ConanFile):
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    def requirements(self):
-        self.requires("rapidjson/1.1.0")
+    def _min_cppstd(self):
+        return 14
 
-    def validate(self):
-        minimal_cpp_standard = "14"
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, minimal_cpp_standard)
-        minimal_version = {
+    @property
+    def _compilers_minimum_version(self):
+        return {
             "gcc": "5",
             "clang": "4",
             "apple-clang": "8",
             "Visual Studio": "15"
```

Comment:
```suggestion
            "Visual Studio": "14",
            "msvc": "190",
```

Reply:
quite disagree since it goes against the original author's intention about the minimal version, but ok.